src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -140,8 +142,7 @@ def get_traits_json():
   """
   try:
     traits_query = query_phenotype_metadata()
-    traits_json = [ trait.to_json() for trait in traits_query]
-    traits_json = convert_joined_query_tojson(traits_query)
+    traits_json = [ trait.to_json_with_values() for trait in traits_query]
   except Exception:
```

Comment:
we need to differentiate empty result set from errors to the end-user. Let's discuss after this cycle.